.changeset/config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
For reviewers: here is the schema: https://unpkg.com/@changesets/config@1.5.0/schema.json